client/src/app/components/NetworkSwitcher.scss

```
@@ -68,7 +68,9 @@
   }
 
   .protocols {
-    display: flex;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```

We need another column here
```suggestion
    grid-template-columns: 1fr 1fr 1fr;
```